lte/gateway/python/mypy.ini

```diff
@@ -0,0 +1,21 @@
+[mypy]
+python_version = 3.7
+ignore_missing_imports = True
+show_error_codes = True
+warn_unused_configs = True
+ignore_errors = False
+exclude = (?x)(
+    ^python/magma/pipelined/ | 
+    ^python/magma/enodebd/ | 
+    ^python/magma/subscriberdb/protocols/diameter/ | 
+    ^python/magma/kernsnoopd/ | 
+    ^python/precommit.py$ | 
+    ^python/scripts/agw_health_cli.py$ |
+    ^python/load_tests.common.py$ |
+    ^python/scripts/generate_oai_config.py$ |
+    ^python/integ_tests/s1aptests/s1ap_utils.py$ |
+    ^python/integ_tests/s1aptests/test_attach_detach_non_nat_dp_ul_tcp.py$ |
+    ^python/integ_tests/s1aptests/util/traffic_messages.py$ |
```

Comment:
Except the integration tests, which are still executed on 3.5 Might make sense to keep them excluded for the moment.